src/pages/Home/Hero.jsx

```diff
@@ -1,27 +1,47 @@
-import { Component } from "react";
+import { memo } from "react";
```

Comment:
I'm not a big fan of using React's `memo` except for very specific types of performance optimizations (generally involving components that handle expensive-to-initialize internal state such as a WebGL canvas, which live inside other components that need to rerender in a tight loop). Using it in these components shouldn't be necessary since (1) they should be cheap to render and (2) they live on a mostly static homepage where their parent shouldn't be rerendering at high frequency. If either of those claims is false then we should fix that issue rather than masking it by memoizing.

Reply:
agreed.  memo is avoidable.  collin please resolve